parser.pegjs

```diff
@@ -60,10 +69,36 @@ pluralKey
   = id
   / '=' d:digits { return d; }
 
-functionParams = _ ',' _ p:id _ { return p; }
+functionParams
+  = p:functionParamsDefault* ! { return options.strictFunctionParams; } { return p; }
+  / p:functionParamsStrict* & { return options.strictFunctionParams; } { return p; }
+
+functionParamsStrict = _ ',' p:paramStrict { return p; }
+
+functionParamsDefault = _ ',' _ p:paramDefault _ { return p.replace(/^[ \t\n\r]*|[ \t\n\r]*$/g, ''); }
```

Comment:
Because `_` is just `[ \t\n\r]` and trim is `[\s]`, which is `[ \f\n\r\t\v\u00a0\u1680\u2000-\u200a\u2028\u2029\u202f\u205f\u3000\ufeff]`, and I wanted to keep it the same.

Changing them both to \s would probably be fine.